### PR TITLE
add test for null url to ComponentUrl tests for clarity of intention

### PR DESCRIPTION
**Pull request by TheMallen**
This PR simply adds a test to `ComponentUrl` so that it's behaviour on a null url is clearer to consumers.

@Shopify/tnt 
@qq99 
@GoodForOneFare

### CODE REVIEW

File: test/javascripts/component_url_test.coffee [hunk 1]
```diff
@@ -1,5 +1,9 @@
 describe 'ComponentUrl', ->
   describe 'constructor', ->
+    it 'uses current location when given not given a url', ->
```
Comment (arapehl) [inline test/javascripts/component_url_test.coffee]: >given not given
?

**Comment by qq99** [comment]
👍

File: test/javascripts/component_url_test.coffee (hunk 1 above)
Comment (TheMallen) [inline test/javascripts/component_url_test.coffee]: Oops/

**Comment by GoodForOneFare** [comment]
👍 🚢 🐑
